File: jTierra/src/main/java/com/cristiano/jTierra/instructions/InstructionBuilderTierra.java
package com.cristiano.jTierra.instructions;

import com.cristiano.alife.consts.ALifeConsts;
import com.cristiano.alife.instructions.io.MalInstruction;
import com.cristiano.alife.world.InstructionBuilderBase;
import com.cristiano.alife.world.IWorld;

public class InstructionBuilderTierra extends InstructionBuilderBase {

	public InstructionBuilderTierra(IWorld m) {
		super(m);
	}

	public void build(IWorld m) {
		super.build(m);
		addInstruction(INSTR_MAL, new MalInstruction(mundo));
	}

}

File: netAvida/netAvida.Tierra/InstructionBuilderTierra.cs
using netAvida.backend.interfaces;
using netAvida.Backend.instructions;
using netAvida.Backend.instructions.impl;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace netAvida.Tierra
{
    public class InstructionBuilderTierra: InstructionBuilderBase
    {
        public InstructionBuilderTierra(IWorld m):base(m)
        {
        }

        public override void build(IWorld m)
        {
            base.build(m);
            addInstruction(INSTR_MAL, new MalInstruction(mundo));
        }
    }
}
